mqtt/mqtt-bridge/src/client.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::default_trait_access)] // Needed because mock! macro violates
```

Comment:
Putting this above the macro didn't work.

Reply:
Ok, I fixed this for `Default`, it seems like we need something more reliable